docs/reference/method/MongoDBModelCollectionInfo-getInfo.txt

```diff
@@ -0,0 +1,57 @@
+=========================================
+MongoDB\\Model\\CollectionInfo::getInfo()
+=========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Model\\CollectionInfo::getInfo()
+
+   Returns additional information about the collection.
+
+   .. code-block:: php
+
+      function getInfo(): array
```

Comment:
I had a thought about how this and `getOptions` cast the returned value to an array and how that might be inconsistent is a nested value was a document and was reported as a stdClass. I then remembered that ListCollections doesn't support a `typeMap` option at all and applies its own typeMap such that root and embedded docs become PHP arrays. So there's no reason to be concerned about inconsistency -- and I also don't worry about the MongoDB server returning a value that would be ambiguous (e.g. document with sequential numeric keys).